html/pfappserver/lib/pfappserver/Form/Config/Source/Kerberos.pm

```diff
@@ -25,14 +25,15 @@ has_field 'host' =>
    element_class => ['input-small'],
    element_attr => {'placeholder' => '127.0.0.1'},
   );
-has_field 'realm' =>
+has_field 'authenticate_realm' =>
```

Comment:
Doesn't seem like your upgrade script handles that parameter rename

Reply:
Done